docs/src/main/asciidoc/building-my-first-extension.adoc

```diff
@@ -28,7 +28,7 @@ To complete this guide, you need:
 * less than 30 minutes
 * an IDE
 * JDK 1.8+ installed with `JAVA_HOME` configured appropriately
```

Comment:
I changed it back to 1.8+ as we are still supporting 1.8 and all the other guides reference 1.8. Better update them all at once (and make it a variable in passing).